docs/how-to/monitor-your-backups.md

```diff
@@ -290,6 +291,75 @@ If you have any issues with the integration, [please contact
 us](https://torsion.org/borgmatic/#support-and-contributing).
 
 
+## Pushover hook
+
+<span class="minilink minilink-addedin">New in version 1.9.0</span>
+[Pushover](https://pushover.net) makes it easy to get real-time notifications 
+on your Android, iPhone, iPad, and Desktop (Android Wear and Apple Watch, too!)
```

Comment:
Nit: Add period at the end of the sentence.

Reply:
https://github.com/borgmatic-collective/borgmatic/commit/7eb19cb0a7747b5062b9ed3f8ca1f9e67a4efe20